### PR TITLE
Relay Modern

### LINKED ISSUE #2: Does "Relay Modern" change anything?
I know [relay modern](https://facebook.github.io/relay/docs/relay-modern.html) is backwards-compatible ... just asking if new things are possible?

### PR DESCRIPTION
**Pull request by sibelius**
fix #2

### CODE REVIEW

**Comment by johntran** [comment]
You're the man :)

File: package.json [hunk 1]
```diff
@@ -51,6 +52,8 @@
     "clear": "node node_modules/react-native/local-cli/cli.js start --reset-cache",
     "lint": "eslint src",
     "lint:staged": "lint-staged",
+    "relay": "relay-compiler --src ./src --schema data/schema.graphql",
+    "relay:watch": "relay-compiler --src ./src --schema data/schema.graphql --watch",
```
Comment (trevordmiller) [inline package.json]: You can do `"relay:watch": "yarn relay -- --watch"` to pass an argument to another yarn script

File: .babelrc [hunk 2]
```diff
@@ -4,9 +4,10 @@
   "comments": false,
   "presets": [
     "react-native",
+    "react-native-stage-0/decorator-support"
   ],
   "plugins": [
-    "./plugins/babelRelayPlugin"
+    ["relay", {"compat": true, "schema": "data/schema.json"}],
```
Comment (trevordmiller) [inline .babelrc]: Can just be ["relay"] unless compatibility mode is needed
https://facebook.github.io/relay/docs/babel-plugin-relay.html

Comment (sibelius) [inline .babelrc]: I'm reaching this issue here: https://github.com/facebook/relay/issues/1665

I'll try without `compat` mode

File: package.json (hunk 1 above)
Comment (sibelius) [inline package.json]: upgraded tks

File: .babelrc (hunk 2 above)
Comment (trevordmiller) [inline .babelrc]: Gotcha, didn't know about that

**Pull request merged**